examples/gcoap/README.md

```diff
@@ -53,6 +67,18 @@ CLI output:
     </>;title="General Info";ct=0,</time>;if="clock";rt="Ticks";title="Internal Clock";ct=0;obs,</async>;ct=0
 
 
+## Other available CoAP implementations and applications
+
+RIOT is compatible with other CoAP implementations with associated test
```

Comment:
"compatible" is kind of general. Might be clearer to say: 

RIOT also provides package imports and test applications for other CoAP implementations: